### ISSUE TITLE
Does Flashinfer support 8-bit attention calculation?

### ISSUE THREAD

**Issue by Juelianqvq** [flashinfer-ai/flashinfer#502]
I mean without dequantizing 8-bit kv-cache to fp16

**Comment by zhyncs** [comment]
This should be in the plan. ref https://github.com/sgl-project/sglang/pull/1286#issuecomment-2323725290

**Comment by yzh119** [comment]
@Juelianqvq  The feature was deferred because the wheel binary size explosion, I'll work on this after #507 was merged.
